src/components/pages/TeachersPage/index.tsx

```diff
@@ -20,6 +20,215 @@ interface StudentChat {
   startTime: string;
 }
 
+const EXAMPLE_CHATS = [
+  {
+    chatId: '8DmBxTmt0Q_iUbbxAAAn#Ua_hGB2pSD-6CGvdAAAP',
+    studentPair: [
+      {
+        realName: 'Student 1797',
```

Comment:
nit: I would move this into its own file

Reply:
I just removed it as I just intended to use it to display the layout, but if you find it helpful for dev purposes I can leave it as a seperate file

Reply:
ah gotcha, no, deleting is fine, just wasn't sure if this was supposed to be a part of the finished product

Reply:
@karlrez actually I do think we should keep these example chats for dev purposes. Can you put it as its own file? So other devs will realize this file exists, I think it should go under the TeachersPage folder.   You could leave a comment at the top of the file explaining the file isn't used in production but is there for dev purposes so that any developer can create a whole bunch of example chats to test out the UI.